src/DssSpell.t.sol

```diff
@@ -1960,19 +2041,46 @@ contract DssSpellTest is DSTest, DSMath {
         }
         // dart max amount of DAI
         (,,uint256 spot,,) = vat.ilks(_ilk);
-        vat.frob(_ilk, address(this), address(this), address(this), int(amount), int(mul(amount, spot) / rate));
+        vat.frob(_ilk, address(this), address(this), address(this), int256(amount), int256(mul(amount, spot) / rate));
         hevm.warp(block.timestamp + 1);
         jug.drip(_ilk);
-        assertEq(flip.kicks(), 0);
+        assertEq(clip.kicks(), 0);
         if (_checkLiquidations) {
-            cat.bite(_ilk, address(this));
-            assertEq(flip.kicks(), 1);
+            dog.bark(_ilk, address(this), address(this));
+            assertEq(clip.kicks(), 1);
         }
 
         // Dump all dai for next run
         vat.move(address(this), address(0x0), vat.dai(address(this)));
     }
 
+    function testCollateralIntegrations() public {
```

Comment:
This function exists on line 2432 as well, but commented out. Maybe remove that one. Also, slight nit, but I think this function should come below all the `checkXXXIntegration` variants.

Reply:
adjusted.